plugins/woocommerce/src/Internal/Admin/Orders/MetaBoxes/CustomMetaBox.php

```diff
@@ -0,0 +1,92 @@
+<?php
+/**
+ * Meta box to edit and add custom meta values for an order.
+ */
+
+namespace Automattic\WooCommerce\Internal\Admin\Orders\MetaBoxes;
+
+/**
+ * Class CustomMetaBox.
+ */
+class CustomMetaBox {
+
+	/**
+	 * Constructor, adds necessary hooks.
+	 */
+	public function __construct() {
+		add_filter( 'postmeta_form_keys', array( $this, 'order_meta_keys_autofill' ), 10, 2 );
+	}
+
+	/**
+	 * Renders the meta box to manage custom meta.
+	 *
+	 * @param \WP_Post|\WC_Order $order_or_post Post or order object that we are rendering for.
+	 */
+	public function output( $order_or_post ) {
+		if ( is_a( $order_or_post, \WP_Post::class ) ) {
+			$order = wc_get_order( $order_or_post );
+		} else {
+			$order = $order_or_post;
+		}
+
+		$metadata         = $order->get_meta_data();
+		$metadata_to_list = array();
+		foreach ( $metadata as $meta ) {
+			$data = $meta->get_data();
+			if ( is_protected_meta( $data['key'], 'order' ) ) {
+				continue;
+			}
+			$metadata_to_list[] = array(
+				'meta_id' => $data['id'],
+				'meta_key' => $data['key'],
+				'meta_value' => $data['value'],
+			);
+		}
+		$this->render_custom_meta_form( $metadata_to_list, $order );
+	}
+
+	/**
+	 * Helper method to render layout and actual HTML
+	 *
+	 * @param array     $metadata_to_list List of metadata to render.
+	 * @param \WC_Order $order Order object.
+	 */
+	private function render_custom_meta_form( array $metadata_to_list, \WC_Order $order ) {
+		?>
+		<div id="postcustomstuff">
+			<div id="ajax-response"></div>
+			<?php
+			list_meta( $metadata_to_list );
+			meta_form( $order );
+			?>
+		</div>
+		<p>
+			<?php
+			printf(
+			/* translators: %s: Documentation URL. */
+				esc_html( __( 'Custom fields can be used to add extra metadata to an order that you can <a href="%s">use in your theme</a>.', 'woocommerce' ) ),
+				esc_attr( __( 'https://wordpress.org/support/article/custom-fields/', 'woocommerce' ) )
+			);
```

Comment:
What do you think of:

```suggestion
			printf(
				/* translators: 1: opening documentation tag 2: closing documentation tag. */
				esc_html( __( 'Custom fields can be used to add extra metadata to an order that you can %1$suse in your theme%2$s.', 'woocommerce' ) ),
				'<a href="' . esc_attr__( 'https://wordpress.org/support/article/custom-fields/', 'woocommerce' ) . '">',
				'</a>'
			);
```

This moves the HTML out of the translatable string, and also avoids escaping the HTML (as currently written, we don't actually see a properly rendered link).